src/cgroup.rs

```diff
@@ -241,6 +261,14 @@ impl Cgroup {
         let _ = self.hier.root_control_group().add_task(pid);
     }
 
+    /// Removes a proc from the control group.
+    ///
+    /// Note that this means that the task will be moved back to the root control group in the
+    /// hierarchy and any rules applied to that control group will _still_ apply to the proc.
+    pub fn remove_proc(&self, pid: CgroupPid) {
```

Comment:
Why we move the process to root cgroup instead of its parent cgroup?

Reply:
After many tests, it was found that the process could not be moved to the parent control group but could be moved to the root control group. I added the methods move_proc_to_parent and move_task_to_parent to move processes and threads to the parent control group, but kept the method to move to the root control group to ensure that projects using cgroups-rs crate can use remove_proc and remove_task normally.